src/main/java/com/example/LoyaltyCard.java

```diff
@@ -0,0 +1,36 @@
+package com.example;
+
+import static com.google.common.base.Preconditions;
+
+
+/**
+ * Each customer has his own card that serves as an additional identifier.
+ */
+public final class LoyaltyCard {
+    /**
+     * Customer's card id.
```

Comment:
Is it an ID or a number? What's the difference? Why isn't the field named `id` then?